opentelemetry-sdk/tests/metrics/integration_test/test_histogram_export.py

```diff
@@ -0,0 +1,85 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+from opentelemetry.metrics import set_meter_provider
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import InMemoryMetricReader
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+in_memory_metric_reader = InMemoryMetricReader()
+
+provider = MeterProvider(
+    resource=Resource.create({SERVICE_NAME: "otel-test"}),
+    metric_readers=[in_memory_metric_reader],
+)
+set_meter_provider(provider)
+
+meter = provider.get_meter("my-meter")
+
+histogram = meter.create_histogram("my_histogram")
+counter = meter.create_counter("my_counter")
```

Comment:
Can you move this into setUp so if more test cases are added, it will be automatically reset